packages/ember-data/lib/serializers/json-api-serializer.js

```diff
@@ -26,6 +26,27 @@ export default JSONSerializer.extend({
   isNewSerializerAPI: true,
 
   /*
+    @method _normalizeDocumentHelper
+    @param {Object} documentHash
+    @return {Object}
+    @private
+  */
+  _normalizeDocumentHelper: function(documentHash) {
```

Comment:
Just curious, why introduce `_normalizeDocumentHelper` instead of calling `_normalizeResponse` directly?